renderer.js

```diff
@@ -6,6 +6,101 @@ the purpose of this file is to hold these objects and to make sure they are corr
 import * as THREE from 'three';
 import {EffectComposer} from 'three/examples/jsm/postprocessing/EffectComposer.js';
 import {minFov} from './constants.js';
+import {init, vec3, vec4, mat4} from 'glmw';
+
+// https://stackoverflow.com/a/62544968/3596736
+// Adding 'support' for instanceof Proxy:
+(() => {
+  var proxyInstances = new WeakSet();
+
+  // Optionally save the original in global scope:
+  window.OriginalProxy = Proxy;
+
+  window.Proxy = new Proxy(Proxy, {
+    construct(target, args) {
+      var newProxy = new window.OriginalProxy(...args);
+      proxyInstances.add(newProxy);
+      return newProxy;
+    },
+    get(obj, prop) {
+      if (prop === Symbol.hasInstance) {
+        return instance => {
+          return proxyInstances.has(instance);
+        };
+      }
+      return Reflect.get(...arguments);
+    },
+  });
+})();
+
+init().then(() => {
+  THREE.Matrix4.prototype.multiplyMatrices = (function() {
+    const mat4a = mat4.create();
+    const mat4b = mat4.create();
+    const mat4c = mat4.create();
+    const mat4aView = mat4.view(mat4a);
+    const mat4bView = mat4.view(mat4b);
+    const mat4cView = mat4.view(mat4c);
+
+    return function multiplyMatrices() {
+      if (Array.isArray(this.elements)) {
```

Comment:
This replacement should probably happen at Object3D create time.

In which case we might need a truly vendorized THREE.js, if we don't want to create extra throwaway `.element` arrays.

Reply:
Forked threejs lib and modified.
Now can use `npm install https://github.com/gonnavis/three.js/tarball/WebaverseR133D1` to install modified threejs version.

But wasm must be initialized asynchronously prevent me initiate `elements` with wasm at start.